native-sql-engine/core/src/main/scala/com/intel/oap/execution/ColumnarBroadcastHashJoinExec.scala

```diff
@@ -59,7 +59,7 @@ case class ColumnarBroadcastHashJoinExec(
     nullAware: Boolean = false)
     extends BaseJoinExec
     with ColumnarCodegenSupport
-    with ShuffledJoin {
+    with ColumnarShuffledJoin {
```

Comment:
I think we can still let it extend ShuffledJoin. Need verify in compile and test.